docs/source/models/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -330,6 +330,61 @@ Note:
   - The draws were also looked at in terms of input data - MI incidence, SBP mean value, etc to ensure that it was also reasonably close to the average for `input data <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Draw_Correlation.ipynb>`_ 
 
 
+**Draw Testing:**
+
+.. list-table:: CVD simulation model population parameters
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+     - Note
+   * - Population size
+     - 1 seed
+     - 
+   * - Number of draws
+     - 50
+     - 
+   * - Cohort type
+     - Closed
+     - 
+   * - Age start
+     - 5 years
+     - Minimum age at initialization was chosen to have youngest simulants be 25 at the end. Ages 5-25 will be modeled but not observed. 
+   * - Age end
+     - 125 years
+     - Maximum age at initialization
+   * - Sex restrictions
+     - None 
+     - 
+
+**Seed Testing:**
+
+.. list-table:: CVD simulation model population parameters
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+     - Note
+   * - Population size
+     - 50 seeds
+     - 
+   * - Number of draws
+     - 1
```

Comment:
I think I'd suggest running maybe 5 draws or so here - the pattern by seed can vary somewhat substantially by draw